ci-operator/step-registry/openshift/e2e/cert-rotation/prepare-for-shutdown/openshift-e2e-cert-rotation-prepare-for-shutdown-commands.sh

```diff
@@ -242,6 +257,12 @@ oc adm wait-for-stable-cluster --minimum-stable-period=1m --timeout=30m
 source /usr/local/share/cert-rotation-functions.sh
 prepull-tools-image-for-gather-step
 
+# Set ClientAliveInterval
```

Comment:
After modifying the SSH configuration (/etc/ssh/sshd_config), you need to restart the SSH service for the changes to take effect.

Reply:
Oh, good catch, thanks